dsp/modules/together_client.py

```diff
@@ -0,0 +1,171 @@
+import logging
```

Comment:
DSPy already supports a Together client via hf_client. this file (and related changes can be removed)